app/src/main/java/com/kms/cura/view/RegDoc_AccountFragment.java

```diff
@@ -0,0 +1,139 @@
+package com.kms.cura.view;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.Toast;
+
+import com.kms.cura.R;
+import com.kms.cura.utils.InputUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class RegDoc_AccountFragment extends Fragment implements View.OnClickListener, TextWatcher {
+
+    EditText edtPhone, edtEmail, edtPwd, edtRePwd;
+    Button btnRegister;
+    ImageButton btnBack;
+
+    public RegDoc_AccountFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View myFragmentView = inflater.inflate(R.layout.fragment_reg_doc__account, container, false);
+        edtPhone = setupEditText(myFragmentView, R.id.edtPhone, this);
+        edtEmail = setupEditText(myFragmentView, R.id.edtEmail, this);
+        edtPwd = setupEditText(myFragmentView, R.id.edtPwd, this);
+        edtRePwd = setupEditText(myFragmentView, R.id.edtRePwd, this);
+        btnRegister = (Button) myFragmentView.findViewById(R.id.btnRegister);
+        btnRegister.setOnClickListener(this);
+        btnBack = (ImageButton) myFragmentView.findViewById(R.id.btnRegisterBack);
+        btnBack.setOnClickListener(this);
+        // reform data
+        Bundle bundle = getArguments();
+        edtPhone.setText(bundle.getString(RegisterDoctorActivity.PHONE, ""));
+        edtEmail.setText(bundle.getString(RegisterDoctorActivity.EMAIL, ""));
```

Comment:
You should extract 
`Bundle bundle = getArguments();
edtPhone.setText(bundle.getString(RegisterDoctorActivity.PHONE, ""));
edtEmail.setText(bundle.getString(RegisterDoctorActivity.EMAIL, ""));` into a method called reformData to avoid encumber onCreateView()